### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cameron-keene**
Added table of contents to readme and information about the LICENSE

### CODE REVIEW

**Comment by WerWolv** [comment]
Thank you!

I don't mind adding a section about the license, however the table of contents looks kinda weird in my opinion. Removing the `*` in front of every link and centering the list might help? Not sure

File: README.md [hunk 1]
```diff
@@ -196,3 +221,9 @@ All dependencies that aren't bundled, can be installed using the dependency inst
 - Thanks to btzy for [nativefiledialog-extended](https://github.com/btzy/nativefiledialog-extended)
 - Thanks to danyspin97 for [xdgpp](https://sr.ht/~danyspin97/xdgpp)
 - Thanks to all other groups and organizations whose libraries are used in ImHex
+
+
+### License
+
+- ImHex is available under the GNU General Public License. Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.
```
Comment (iTrooz) [inline README.md:228]: I propose to explicitly say the license is the GPL **2**

**Comment by KillyMXI** [comment]
Seems like TOC formatting was messed up and it was supposed to be just another unordered list but can't be parsed as such by GitHub.
In that case I would use dashes for consistency (and fix spacing of course).

Compact inline TOC might be not a bad choice for a sigle-level list (indented entries would have to go if it were up to me) but that would require more deliberate formatting compared to what is provided in the PR.

**Comment by WerWolv** [comment]
Sorry for ignoring this for so long. Closing it though since I've since refactored the readme and moved this information to the INSTALL.md file (including some of the proposed changes from you). Thank you though!

**Pull request closed without merging**